### ISSUE TITLE
module 'bluepy.btle' has no attribute 'BTLEDisconnectError'

### ISSUE THREAD

**Issue by pattyland** [k0rventen/avea#3]
In the last time this error pops up more often:

```
[...]
File "/config/custom_components/avea/light.py", line 84, in turn_off
    self._light.set_brightness(0);
  File "/config/deps/lib/python3.7/site-packages/avea/avea.py", line 83, in set_brightness
    if self.connect():
  File "/config/deps/lib/python3.7/site-packages/avea/avea.py", line 61, in connect
    except bluepy.btle.BTLEDisconnectError:
AttributeError: module 'bluepy.btle' has no attribute 'BTLEDisconnectError'
```

**Comment by k0rventen** [comment]
Hey @pattyland ! I've looked at your issue and I don't understand exactly what is going wrong in the except statement.. But to workaround the issue for now I've patched avea. 

I'll dig into it more next week, but for now everything should work as expected. 
I'll leave the issue open for now, and will close it when a proper fix is up. 

Tell me if with the new 1.2.8 version you encounter the same error !
